crates/typst-pdf/src/convert.rs

```diff
@@ -81,6 +81,22 @@ fn convert_pages(gc: &mut GlobalContext, document: &mut Document) -> SourceResul
                 typst_page.frame.height().to_f32(),
             );
 
+            if typst_page.bleed != Sides::splat(Abs::zero()) {
+                settings = settings
+                    .with_bleed_box(Rect::from_xywh(
+                        0.0,
+                        0.0,
+                        typst_page.frame.width().to_f32(),
+                        typst_page.frame.height().to_f32(),
+                    ))
```

Comment:
I think writing this is a bit redundant as it will default to the MediaBox anyway.